lib/AmdMainTemplatePlugin.js

```diff
@@ -1,43 +1,55 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
-*/
-var ConcatSource = require("webpack-sources").ConcatSource;
+ MIT License http://www.opensource.org/licenses/mit-license.php
+ Author Tobias Koppers @sokra
+ */
 
-function AmdMainTemplatePlugin(name) {
-	this.name = name;
+"use strict";
+
+const ConcatSource = require("webpack-sources").ConcatSource;
+
+class AmdMainTemplatePlugin {
+	constructor(name) {
+		this.name = name;
+	}
+
+	apply(compilation) {
+		const mainTemplate = compilation.mainTemplate;
```

Comment:
I know all const is a preference for some, but can you change (like the other PR I commented on) const to let for all object-based primitives (object, array), and non-object based primitives can use const as long as they don't change.